app/connectors/CisConnector.scala

```diff
@@ -28,25 +28,25 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.ExecutionContext
 
 @Singleton
-class IncomeTaxCisConnector @Inject()(config: FrontendAppConfig, httpClient: HttpClientV2)
+class CisConnector @Inject()(config: FrontendAppConfig, httpClient: HttpClientV2)
```

Comment:
unless there is very specific reason to refactor this? we should try to avoid such changes in PR, as this creates distraction. It would be better to track these under separate ticket with some reasoning behind it.

Reply:
Currently CIS follows the wrong naming convention completey

Reply:
Thanks Matt, i have approved the PR, my suggestion was to do these changes separately so that it would be easy to review PR. Marking this as resolved